### DISCUSSION TITLE
PCA plot

### DISCUSSION THREAD

**Discussion by gsk7757** [biostars/biostar-handbook-issues#307] [Q&A]
Hello everyone, 
I tried to run RScript code/create_pca.r to create a PCA plot. However, the code returns the below error. 
Error: Counts file, Design file and the number of samples must be specified at the commandline
Execution halted

I have mentioned counts, design, and number of samples in the command line. Here is how I have mentioned
# How to run?
# Rscript summary_plots.r <counts_file> <design_file> <number_of_samples>
# Example: Rscript summary_plots.r counts.txt design.txt 6
# Design file example:
#	Sample	Condition
#	C1dedup	control
#	C2dedup	control
#	C3dedup	control
#	E1dedup	experiment
#	E2dedup	experiment
#	E3dedup	experiment

Please help.

**Comment by ialbert** [comment]
I would recommend following the workflows in the Biostar Workflows

https://www.biostarhandbook.com/books/workflows/

there is an entire section of RNA-Seq methods

All the analysis code has been rewritten in a more elegant and robust manner. 

The "RNA-Seq by Example" book will be soon (within weeks) be rewritten to use the same codebase as the Workflows book.

**Reply by gsk7757** [reply]
Thank you. I will follow the workflow then. I was actually following the book RNA-seq by Example.
